config/analytics.yml

```diff
@@ -399,6 +399,23 @@ shared:
     - selected
     - updated_at
     - refused
+  sites:
```

Comment:
could we move this and the blocklist change to the next migration pr? as that is when we'll stop creating temp sites, and officially want to send sites, also we can remove course_options site id/temp id

Reply:
🆗